azure-communication-ui/build.gradle

```diff
@@ -51,11 +51,16 @@ buildscript {
             return rootProject.hasProperty("disableTaskRootCheck") && rootProject.getProperty("disableTaskRootCheck")
         }
 
-        compose_version = '1.3.1'
+        androidx_lifecycle_runtime_ktx_version = '2.3.1'
+        androidx_activity_compose_version = '1.5.1'
+        compose_version = '1.2.1'
+        compose_material3_version = '1.0.0-alpha12'
```

Comment:
for which component we are using material version? I think we will add if required in futre.

Reply:
Sorry. Removed this from dependency but did not remove from version gradle script. Removing this.